a10_octavia/controller/worker/controller_worker.py

```diff
@@ -31,6 +31,7 @@
 from octavia.common import constants
 from octavia.common import base_taskflow
 from octavia.common import exceptions
+from a10_octavia.common import data_models
```

Comment:
can we have imports in sections in following sequence:

#1 Python standard library imports
#2 Installed python modules imports
#3 octavia imports 
#4 a10-octavia imports
 
Within sections, a space is expected, and each section is alphabetically sorted

Reply:
@afrin-chakure  @NehaKembalkarA10  follow the same from now onwards

Reply:
Agreed yes. However, that is out of scope for this PR as it is meant to get bring stein to the head. A refactor will be needed to fix imports. Please add a task to the JIRA board under refactor to capture this.